.github/workflows/binaries-analysis.yml

```diff
@@ -0,0 +1,44 @@
+name: "Binaries Analysis"
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [ "main" ]
+  schedule:
+    - cron: '21 17 * * 0'
```

Comment:
💡☕️ Idea share: Is this very specific for some reasons? i.e. So `* * * * *` means `every minute` of `every hour` of `every day` of `every month` and `every day` of the week. Just checking, 21st min of every 17 hours  or every day to very month and only run on Sundays. Thank you for this.

Reply:
Nope, just followed the default settings for Cron job, it runs every Sunday at 5:21. If you want to change this, please let me know or do you think this schedule cron is not necessary please let me know. Thank you.

Reply:
❤️🙏 thank you for the reply, I ask because see line 5 - we do this binary check for pull-request, push to main branch. So, running on Sunday is just one time thing which is well thought then sounds good to me, please do resolve this conversation as I have approved it. ☕️😊 thanks heaps.

Reply:
Thank you, closing this conversation.